src/appengine/handlers/cron/fuzzer_and_job_weights.py

```diff
@@ -136,7 +136,7 @@ def _coverage_formatter(query_format, dataset):
 # healthy fuzzers are expected to have some crashes.
 CRASH_SPECIFICATION = QuerySpecification(
     query_format=GENERIC_QUERY_FORMAT.format(
-        field_name='crash_count', min_weight=0.70),
+        field_name='crash_count', min_weight=0.50),
```

Comment:
I'd drop both this and oom as well a bit more

Reply:
It depends how much fuzzers is crashing here, decreasing this can hurt real fuzzers which find some crashes from time to time.

Reply:
Since we scale linearly based on how often the crashes are occurring it might be reasonable to go lower. I'm not sure what the right value would be here, but it could be lowered more.